src/utils/hederaUtils.js

```diff
@@ -240,6 +227,28 @@ module.exports = class HederaUtils {
     await this.generateED25519(client, num, startup, logger);
   }
 
+  static async createAccount(client, wallet, accountNum, logger, startup) {
+    const tx = await new HederaSDK.AccountCreateTransaction()
+      .setKey(
+        HederaSDK.PublicKey.fromString(
+          wallet._signingKey().compressedPublicKey
+        )
+      )
+      .setInitialBalance(HederaSDK.Hbar.fromTinybars(10000000000000))
+      .execute(client);
+    if (startup) {
+      logger.log(
```

Comment:
3rd and 4th occurrence of this log.
Could also just create a method logAccountDetails(accountID)
AccountId can be the already formatted account i.e. `0.0.${accountNum}` or `getReceipt.accountId.toString()`

Reply:
Thanks, I've extracted this in separate method.